demisto_sdk/commands/prepare_content/integration_script_unifier.py

```diff
@@ -379,18 +381,25 @@ def check_api_module_imports(script_code: str) -> Dict[str, str]:
         }
 
     @staticmethod
-    def insert_module_code(script_code: str, import_to_name: Dict[str, str]) -> str:
+    def insert_module_code(
+        script_code: str, import_to_name: Dict[str, str], content_path: Path
+    ) -> str:
         """
         Inserts API module in place of an import to the module according to the module name
         :param script_code: The integration code
         :param import_to_name: A dictionary where the keys are The module import string to replace
         and the values are The module name
+        :param content_path: The path to the content repo
```

Comment:
explain why this variable is passed (one would wonder why we dont just call `get_content_path` in here)